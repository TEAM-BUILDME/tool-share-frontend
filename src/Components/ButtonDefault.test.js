import React from 'react';
import { create } from 'react-test-renderer';

import ButtonDefault from './ButtonDefault';

describe('Default Button component', () => {
	test('matches the snapshot', () => {
		const button = create(<ButtonDefault />);
		expect(button.toJSON()).toMatchSnapshot();
	});
});
