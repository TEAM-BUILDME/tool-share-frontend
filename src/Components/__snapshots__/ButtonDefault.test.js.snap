// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Button component matches the snapshot 1`] = `
<button
  onClick={[Function]}
/>
`;
